fly/integration/watch_test.go

```diff
@@ -121,6 +146,10 @@ var _ = Describe("Watching", func() {
 		It("Watches the given build id", func() {
 			watch("--build", "3")
 		})
+
+		It("Watches the given build id with timestamp prefixed", func() {
```

Comment:
We can do without these additional tests (this + the rest in the file), since these flags don't really affect each other.